appscale/tools/parse_args.py

```diff
@@ -216,6 +216,25 @@ def add_allowed_flags(self, function):
         choices=self.ALLOWED_GCE_INSTANCE_TYPES,
         help="the Google Compute Engine instance type to use")
 
+      # Microsoft Azure specific flags
```

Comment:
I just added validation for the Azure specific params in the validate_infrastructure_flags() method